x-pack/plugins/security_solution/public/detections/components/alerts_table/actions.tsx

```diff
@@ -844,6 +849,7 @@ export const sendAlertToTimelineAction = async ({
       },
       to,
       ruleNote: noteContent,
+      ruleAuthor,
```

Comment:
While desk testing, I wasn't able to reproduce the race condition in `main` where the investigation guide note could be displayed as the `elastic` user:

<img width="1703" alt="note-from-guide" src="https://user-images.githubusercontent.com/4459398/193330385-647d6117-5242-4e95-9de3-b2d68c54ff00.png">

_Above: In `main` the investigation guide note is attributed to the logged-in user, `andrew`_

, however I was able to verify that before the fix, the note is attributed to the `elastic` user in Redux:

<img width="1041" alt="redux-main" src="https://user-images.githubusercontent.com/4459398/193330687-d9018c88-bb35-48fd-a394-f33956e3c873.png">

_Above: Redux dev tools in  `main`_

and after the fix, it's attributed to the logged-in user:

<img width="1038" alt="redux-pr-branch" src="https://user-images.githubusercontent.com/4459398/193330949-20b77d64-84d9-4060-bb9f-2b763a1fcbb7.png">

_Above: Redux dev tools in  the PR branch_